test/functional/Java8andUp/src/org/openj9/test/annotation/Test_Annotation.java

```diff
@@ -108,6 +111,19 @@
 					"@java.lang.Deprecated()" },
 			{ "@org.openj9.test.annotation.ParameterAnnotation(value=ParameterAnnotationDefault)" } };
 	static Logger logger = Logger.getLogger(Test_Annotation.class);
+	private static final int javaVersion; // java version number for Java 11+
+	private static final boolean isJava8;
+	private static final int JAVA14 = 14;
+
+	static {
```

Comment:
There's a helper class that handles this already - I think it's called VersionCheck.

Reply:
Updated the PR with `VersionCheck.major()`.
Please have another look.